mx.sulong/mx_sulong.py

```diff
@@ -1111,6 +1115,26 @@ def checkNoHttp(args=None):
                 exit(-1)
             line_number += 1
 
+def genInlineAssemblyParser(args=None, out=None):
+    """generate inline assembly parser and scanner if corresponding grammer is new"""
+    generatedParserDir = _inlineAssemblySrcDir + _inlineAssemblyPackageName.replace('.', '/')
+    generatedParserFile = generatedParserDir + "/Parser.java"
+    generatedScannerFile = generatedParserDir + "/Scanner.java"
+    if not isfile(generatedParserFile) or not isfile(generatedScannerFile) or os.path.getmtime(_inlineAssemblyGrammer) >= os.path.getmtime(generatedParserFile):
+        localCocoJarFile = _suite.dir + "/lib/Coco.jar"
+        if not isfile(localCocoJarFile):
+            jarFileUrls = ["http://www.ssw.uni-linz.ac.at/Coco/Java/Coco.jar"]
```

Comment:
I uploaded the file to https://lafo.ssw.uni-linz.ac.at/pub/sulong-deps/Coco.jar, so we can download it from a `https` connection.